Cargo.toml

```
@@ -518,39 +518,6 @@ solana-program = { path = "sdk/program" }
 solana-zk-sdk = { path = "zk-sdk" }
 solana-zk-token-sdk = { path = "zk-token-sdk" }
 
```

```suggestion

[profile.dev.package.curve25519-dalek]
opt-level = 3

```
Let's bump-up the opt-level here for other crates in the monorepo that depends on the curve25519-dalek crate.